### PR TITLE
fix unit tests

### PR DESCRIPTION
**Pull request by ramenhog**
## Issue Number
https://github.com/cognoma/frontend/issues/115

## Purpose/Implementation Notes
* fix existing unit tests so tests can pass in CircleCI
* updated SauceLabs download url
* removed `karma-coverage` for now (will reevaluate later)

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Functional tests
* ran unit tests locally and verified they all pass

## Checklist

_Put an `x` in the boxes that apply._

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ramenhog** [comment]
@dongbohu this is ready for review

File: test/unit/components/app/app_spec.js [hunk 1]
```diff
@@ -16,15 +17,15 @@ describe('Unit: Component: App', function() {
 
   beforeEach(function() {
     
-    angular.mock.inject(($compile, $rootScope) => {
+    angular.mock.inject(($compile, $rootScope, $httpBackend) => {
```
Comment (dongbohu) [inline test/unit/components/app/app_spec.js]: `$httpBackend` is not being used now. Do you plan to rewrite this test case in the near future?

File: test/unit/on_config_spec.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ describe('UNIT: on_config', function () {
         $_rootScope,
         $_templateCache,
         $_location,
-        _state = "app"; //root component state
+        _state = "app"; //redirected root component state
```
Comment (dongbohu) [inline test/unit/on_config_spec.js]: As far as I know, cognoma doesn't have a JS style guide yet. Feel free to create an issue and enforce one to later make the code easier to read (such as a space after `//`).

**Comment by ramenhog** [comment]
@dongbohu should be good!

**Pull request merged**